src/web/client/dal/fileSystemProvider.ts

```diff
@@ -310,6 +310,128 @@ export class PortalsFS implements vscode.FileSystemProvider {
+
+        const matches: ISearchQueryMatch[] = [];
+        const files = await this.iterateDirectory(WebExtensionContext.rootDirectory);
+
+        // Promises array to store promises for file reads
+        const fileReadPromises = files.map(async (file) => {
+            // Record start time for file processing
+            const startFileTime = Date.now();
+
+            const content = await this.readFile(file);
+
+            // Convert buffer to string and replace windows line endings with unix line endings
+            const text = new TextDecoder().decode(content).replace(/\r\n/g, '\n');
+
+            // Convert windows line endings with unix line endings
+            const pattern = query.pattern.replace(/\r\n/g, '\n');
+
+            const lines = text.split('\n');
+
+            const match: ISearchQueryMatch = {
+                uri: file,
+                ranges: [],
+                preview: text,
+                matches: [],
+            };
+
+            let regex;
+            if (query.isWordMatch) {
+                // \b is a word boundary
+                regex = query.isCaseSensitive ? new RegExp(`\\b${pattern}\\b`) : new RegExp(`\\b${pattern}\\b`, "i");
+            } else if (query.isRegExp) {
+                regex = new RegExp(pattern);
+            } else {
+                regex = query.isCaseSensitive ? new RegExp(pattern) : new RegExp(pattern, "i");
+            }
+
+            if (query.isMultiline) {
+                if (text.includes(pattern)) {
+                    const index = text.indexOf(pattern);
+                    const range = getRangeForMultilineMatch(text, pattern, index);
+                    match.ranges.push(range);
+                    match.matches.push(range);
+                    matches.push(match);
+                }
+            } else {
+                for (let i = 0; i < lines.length; i++) {
+                    if (options.maxResults !== undefined && matches.length > options.maxResults) {
+                        return { matches: matches, limitHit: true };
+                    }
+
+                    const regexMatch = lines[i].match(regex);
+
+                    if (regexMatch) {
+                        regexMatch.forEach((m) => {
+                            const index = lines[i].indexOf(m);
+                            const range = new vscode.Range(i, index, i, index + m.length);
+                            match.ranges.push(range);
+                            match.matches.push(new vscode.Range(i, index, i, index + m.length));
+                            matches.push(match);
+                        });
+                    }
+                }
+            }
+
+            // Record end time for file processing
+            const endFileTime = Date.now();
+
+            WebExtensionContext.telemetry.sendInfoTelemetry(
+                telemetryEventNames.WEB_EXTENSION_SEARCH_TEXT_RESULTS,
+                {
+                    file: file.fsPath,
```

Comment:
can you let me know what is getting logged in filePath? Put an example here

Reply:
Had a discussion with @tyaginidhi, and also gave a thought over this and came to the conclusion that we should not print file path. It won't be of any use.

BTW file path was intended to get structuring of current file where search happened and type of file, but on more thought, it seems of not any use for this feature and more over we should not log Cx data.